build-system/pr-check.js

```diff
@@ -97,6 +98,13 @@ function timedExecOrDie(cmd) {
  * Prints a summary of files changed by, and commits included in the PR.
  */
 function printChangeSummary() {
+  if (process.env.TRAVIS) {
+    console.log('Travis', colors.green('origin/master'), 'baseline:',
+        colors.cyan(gitTravisMasterBaseline()));
+  }
+  console.log(colors.green('master'), 'merge-base:',
+      colors.cyan(gitMergeBaseMaster()));
```

Comment:
Delete this block, since it is already being done 10 lines below:

https://github.com/ampproject/amphtml/blob/a191a565992bf0b2d3e8e947cde763bf1f765296/build-system/pr-check.js#L106-L110

Reply:
Done